platform/security/sts/security-sts-attributequeryclaimshandler/src/test/java/org/codice/ddf/security/claims/attributequery/TestAttributeQueryClient.java

```diff
@@ -116,28 +121,33 @@ protected HttpsURLConnection createHttpsUrlConnection(URL url) {
     private String cannedResponse;
 
     @BeforeClass
-    public static void init() {
+    public static void init() throws InitializationException {
         OpenSAMLUtil.initSamlEngine();
+        XMLObjectProviderInitializer initializer = new XMLObjectProviderInitializer();
```

Comment:
Cleaned this up. Now using the InitializationService.initialize() provided by the opensaml library.